ui-components/src/overlays/Overlay.scss

```diff
@@ -44,7 +44,6 @@
 
 .fixed-overlay__inner {
   transition-property: transform, opacity;
-  transform: translate(0px, 0px);
```

Comment:
This was causing a visual bug in the draggable table (weird, I know) - the best suggestion from forums on the package was to remove this style. Let me know if this will cause any other issues, but removing it didn't seem to cause any changes at all.